test/service/test_drs.py

```diff
@@ -76,6 +76,26 @@ def test_data_object_not_found(self):
         else:
             self.fail()
 
+    def test_drs_logs(self):
```

Comment:
It's good to err on the side of writing more tests but this one is overkill. The code is already covered by existing tests. The biggest risk with log statements is not so much that the log message is incorrect or that the log entry isn't made, it's that there is something wrong with the interpolation. In this case, that's covered by existing tests. I just run the tests and check the emitted log message for correctness and call it a day.